torch/csrc/autograd/functions/convolution_attr.h

```diff
@@ -0,0 +1,105 @@
+#pragma once
+
+#include <Python.h>
+#include "convolution.h"
+#include "torch/csrc/autograd/python_cpp_function_attr.h"
+
+
+namespace torch { namespace autograd {
+
+namespace attributes {
+
+template<class T>
+PyObject* conv_stride(THPCppFunction* self, PyObject* hook)
```

Comment:
I agree. I'd be the best to implement a number of helper templates/macros and use these to implement attributes for all new functions in a few lines. See e.g. [some helpers from `python_function.cpp`](https://github.com/pytorch/pytorch/blob/master/torch/csrc/autograd/python_function.cpp#L715-L748). It should be possible to implement similar ones for each type (e.g. getIntMember, getBoolMember, etc.).